### ISSUE TITLE
Exception while unzipping

### ISSUE THREAD

**Issue by Chris2011** [Chris2011/minifierbeans#82]
There is a problem when controling the progress bar:

````
INFO [org.netbeans.modules.progress.spi.InternalHandle]: Cannot process more work than scheduled. Progress handle with name "Extracting minifierbeans archive" has requested progress to workunit no.745547391 but the total number of workunits is 0. That means the progress bar UI will not display real progress and will stay at 100%.

java.lang.IllegalArgumentException
[catch] at org.netbeans.modules.progress.spi.InternalHandle.progress(InternalHandle.java:329)
	at org.netbeans.api.progress.ProgressHandle.progress(ProgressHandle.java:264)
	at org.netbeans.api.progress.ProgressHandle.progress(ProgressHandle.java:242)
	at io.github.chris2011.netbeans.minifierbeans.Installer.extractArchive(Installer.java:229)
	at io.github.chris2011.netbeans.minifierbeans.Installer.access$400(Installer.java:33)
	at io.github.chris2011.netbeans.minifierbeans.Installer$2.run(Installer.java:164)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1418)
	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:45)
	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:278)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2033)
````

Error while extracting: Cannot decrease process workunit count (-2011422090) to lower value than before (0).

**Comment by Chris2011** [comment]
This error preventing the unzip functionality of the node scripts and results in: #70

**Comment by Chris2011** [comment]
Fixed in release 3.3.1

**Comment by JuergenLaas** [comment]
Hello Chris,

I use Netbeans 12.4 with Openj9-0.26.0 and it works!

But i have noticed if i compress CSS-File with your Plugin.

Terminal of Netbeans 12.4 output follow message:
---------------------------------------------------------------------
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
----------------------------------------------------------------------
